server2/Arcadia.Assistant/Arcadia.Assistant.Notifications.Contracts/Models/NotificationMessage.cs

```diff
@@ -16,5 +17,13 @@ public class NotificationMessage
 
         [DataMember]
         public string ShortText { get; set; } = string.Empty;
+
+        [DataMember]
+        public object CustomData { get; set; } = string.Empty;
+
+        public class MessageCustomData
+        {
+            public string Sender { get; set; } = string.Empty;
```

Comment:
Right, we'd have to do that (KnownTypes and all) for custom data. But, why do you need to set Sender in the first place? It's always ArcadiaAssitant From address or do I miss something? Just put in service configuration

Reply:
Removed